solidity/contracts/token/TBTCToken.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity <0.9.0;
+
+import "./ERC20WithPermit.sol";
+
+contract TBTCToken is ERC20WithPermit {
+    constructor() ERC20WithPermit("TBTC v2 migration", "TBTC") {}
```

Comment:
just double checking.. symbol stays the same as in V1 - TBTC?

Reply:
Correct. But there will be a way to change the name (description). This will be done in a separate PR.